docs/user-guide/configure-security-checklist.md

```diff
@@ -0,0 +1,59 @@
+# Configuring security checklist
+
+During the initial installation of Zowe server-side components, it is necessary for your organization's security administrator to perform a range of tasks that require elevated security permissions. As a security administrator, follow the procedures outlined in this checklist to configure Zowe and your z/OS system to run Zowe with z/OS.
+
+**Tip:** We recommend that you consult with your security administrator who has the necessary permissions to perform the following security configuration procedures. 
+
+## Validate and re-run `zwe init` commands
+
+**Important!** During installation, the system programmer customizes values in the zowe.yaml file. However, due to insufficient permissions of the system programmer, the `zwe init security` command fails without notification. 
+
+<!-- ADD INSTRUCTIONS FOR HOW THE SECURITY ADMIN CHECKS FOR FAILED RUNS OF zwe init. -->
+
+<!-- ADD PROCEDURE FOR RE-RUNNING THE `zwe init security` COMMAND -->
+
+## Initialize Zowe security configurations
+
+Choose from the following methods to initialize Zowe security configurations:
+
+* Configuring with `zwe init security`
+* Configuring with `ZWESECUR` JCL
```

Comment:
I think there is one more option that will be the most popular. Do the changes manually. The following sections describe these manual steps.